h2o-docs/src/product/data-science/gam.rst

```diff
@@ -16,213 +16,136 @@ GAM supports importing and exporting `MOJOs <../save-and-load-model.html#support
-   for **Family**) Specify the Tweedie link power (defaults to 0).
-
--  `theta <algo-params/theta.html>`__: Theta value (equal to 1/r) for use with the negative binomial family. This value must be > 0 and defaults to 0.  
-
--  `solver <algo-params/solver.html>`__: Specify the solver to use (AUTO, IRLSM, L_BFGS, COORDINATE_DESCENT_NAIVE, COORDINATE_DESCENT, GRADIENT_DESCENT_LH, or GRADIENT_DESCENT_SQERR). IRLSM is fast on problems with a small number of predictors and for lambda search with L1 penalty, while `L_BFGS <http://cran.r-project.org/web/packages/lbfgs/vignettes/Vignette.pdf>`__ scales better for datasets with many columns. COORDINATE_DESCENT is IRLSM with the covariance updates version of cyclical coordinate descent in the innermost loop. COORDINATE_DESCENT_NAIVE is IRLSM with the naive updates version of cyclical coordinate descent in the innermost loop. GRADIENT_DESCENT_LH and GRADIENT_DESCENT_SQERR can only be used with the Ordinal family. AUTO (default) will set the solver based on the given data and other parameters.
-
--  `alpha <algo-params/alpha.html>`__: Specify the regularization distribution between L1 and L2. The default value of alpha is 0 when ``solver = 'L-BFGS'``, otherwise it is 0.5.
-
--  `lambda <algo-params/lambda.html>`__: Specify the regularization strength.
-
--  `lambda_search <algo-params/lambda_search.html>`__: Specify whether to enable lambda search, starting with lambda max (the smallest :math:`\lambda` that drives all coefficients to zero). If you also specify a value for ``lambda_min_ratio``, then this value is interpreted as lambda min. If you do not specify a value for ``lambda_min_ratio``, then GAM will calculate the minimum lambda. This option is defaults to false (not enabled).
+-  **scale**: An array specifying the smoothing parameter for GAM. If specified, must be the same length as ``gam_columns``.
 
--  `early_stopping <algo-params/early_stopping.html>`__: Specify whether to stop early when there is no more relative improvement on the training or validation set. This option is defaults to true (enabled).
+-  **keep_gam_cols**: Specify whether to save keys storing GAM columns. This option defaults to ``False`` (disabled).
 
-- `stopping_rounds <algo-params/stopping_rounds.html>`__: Stops training when the option selected for **stopping_metric** doesn't improve for the specified number of training rounds, based on a simple moving average. To disable this feature, specify ``0`` (default). 
+-  **splines_non_negative**: (Applicable for I-spline or ``bs=2`` only) Set this option to ``True`` if the I-splines are monotonically increasing (and monotonically non-decreasing). Set this option to ``False`` if the I-splines are monotonically decreasing (and monotonically non-increasing). If specified, this option must be the same size as ``gam_columns``. Values for other spline types will be ignored. This option defaults to ``True`` (enabled).
```

Comment:
please change:
monotonically decreasing (and monotonically non-increasing)

to
monotonically decreasing (or monotonically non-increasing)